extensions/projection/README.md

```diff
@@ -0,0 +1,139 @@
+# Projection Extension Specification (`proj`)
+
+**Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+This document explains the fields of the STAC Projection (`proj`) Extension to a STAC Item. Here `proj` is short
+for "projection", and not a reference to the use of the PROJ format for the CRS attribute.
+
+The field names defined herein should be added as fields in the Item's Properties object. 
+
+When specified on an Item, the values are assumed to apply to all Assets in that Item.  For example, an Item may have several related Assets each representing a band for the Item, all of which are in the same native CRS, e.g., a UTM Zone.  However, there may also be assets intended for display, like a preview image or thumbnail, that have been reprojected to a different CRS, e.g., Web Mercator, to better accommodate that use case.  This case of differing projections per Asset is not currently handled by this extension.
+
```

Comment:
Other extensions link to the JSON schema and a full example (which I am missing here):

* [Example](examples/example.json)
* [JSON Schema](json-schema/schema.json)

Reply:
they existed, but weren't linked. added links.